cuML/test/quasi_newton.cu

```diff
@@ -152,12 +165,13 @@ TEST_F(QuasiNewtonTest, binary_logistic_vs_sklearn) {
 
   l1 = alpha;
   l2 = 0.0;
-  fx = run(loss_b, devUpload, in, l1, l2, w0.data, z, 0, stream);
+  fx = run(loss_b, devUpload, in, l1, l2, w0.data, z, handle, 0);
```

Comment:
Is there a sync needed after run to make sure `compApprox` works correctly?

Reply:
The sync happens everytime the optimizer evaluates the objective to get fx to the host, where the control flow happens.